src/components/grid/Grid.tsx

```diff
@@ -0,0 +1,45 @@
+import _ from 'lodash';
+import { useEffect, useState } from 'react';
+
+import { Board, SquareTypes, Coordinates } from './SquareTypes';
+import HitboxUnavailable from './HitboxUnavailable';
+import Hitbox from './Hitbox';
+import Wall from './Wall';
+
+type PropsGrid = {
+  board: Board;
+  ws: WebSocket;
+  yourTurn: boolean;
+};
+const Grid: React.FC<PropsGrid> = ({ board, ws, yourTurn }) => {
+  const [grid, setGrid] = useState([<mesh></mesh>]);
+
+  const keys = Object.keys(board).map((key) => Number(key));
+
+  useEffect(() => {
+    const gridNew = [<mesh></mesh>];
+    let indexRow = 0;
+    keys.map((row) => {
+      let indexColumn = 0;
+      board[row].squares.map((square) => {
+        if (square.type === SquareTypes.Wall) {
+          const coordinates: Coordinates = { x: indexColumn, y: indexRow };
+          if (square.isAvailable && yourTurn) {
+            gridNew.push(<Hitbox coordinates={coordinates} ws={ws} />);
```

Comment:
Don't pass the whole 'ws' instance to components. Instead, pass a callback to the components which sends the websocket message